substrate/frame/migrations/src/lib.rs

```diff
@@ -378,6 +391,12 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type Historic<T: Config> = StorageMap<_, Twox64Concat, IdentifierOf<T>, (), OptionQuery>;
 
+	/// Data stored by the pre-upgrade hook of the MBM.
+	#[cfg(feature = "try-runtime")]
+	#[pallet::storage]
+	pub type PreUpgradeBytes<T: Config> =
```

Comment:
It could also be a storage alias - give that it is not a proper storage value but only temporarily for testing.